queries/ralph/highlights.scm

```diff
@@ -0,0 +1,91 @@
+(type_identifier) @type
+(primitive_type) @type.builtin
+(alph_token_id) @type.builtin
+
+(var_declaration) @variable
+(struct_field) @variable
+(arg_def (identifier) @variable)
+(struct_constructor_field (identifier) @variable)
+
+(hex_num) @number
+(integer) @number
+(typed_num) @number
+
+(bool)  @constant.builtin
+(bytes) @constant.builtin
+(address) @constant.builtin
+
+(func_id) @function
+
+(mutable) @keyword
+(annotation)  @keyword
+(annotation (identifier) @keyword)
+(annotation_field (identifier) @variable)
+(annotation_field (identifier) @variable)
+(map_def (identifier) @variable)
+
+"Abstract" @keyword
+"Contract" @keyword
+"TxScript" @keyword
+"Interface" @keyword
+"AssetScript" @keyword
+"struct" @keyword
+"fn" @keyword
+"pub" @keyword
+"let" @keyword
+"while" @keyword
+"for" @keyword
+"if" @keyword
+"else" @keyword
+"emit" @keyword
+"enum" @keyword
```

Comment:
Please check `CONTRIBUTING.md`, some of these keywords should be more specific (e.g., `"enum"` should be `@keyword.type`)